### DISCUSSION TITLE
(DEV) Need some examples for Swift or Kotlin code

### DISCUSSION THREAD

**Discussion by BebekBebek99** [phantom/docs#130] [Help & Support] [answered]
### Summary

Hello!
I am developing a project based on the interaction of a mobile application based on IOS / Android with WEB3. Working with the Solana network and Phantom Wallet, I didn’t see your documentation on how I can connect a phantom wallet (and make transactions) in my application. I've seen documentation and examples on React Native, but I think not everyone uses this language. The essence of my question is whether you have any other documentation or examples of code, projects (on github for example) where I can see how work works on Swift & Kotlin.
So i just need some examples of code with how to connect wallet with my app, how i can sign a simple transactions, how i can get user wallet address at all) because i didn't understand so much in wallet documentation, so small amount of information developers provided.
Thanks in advance for you time!

### Steps to Reproduce

idk, i think give developers more code examples and information how to use their wallet.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @BebekBebek99,

I believe you wrote into our Phantom chat support as well.
​
We do not have documentation specific to Swift or Kotlin. Interfacing a mobile app with the Phantom mobile wallet is achieved either via a mobile web app or deeplinking. So our example code is written with web developers in mind and our [deeplinking documentation](https://docs.phantom.app/phantom-deeplinks/deeplinks-ios-and-android) is language agnostic.

From your message I believe deep linking would be the most appropriate path for you. I would recommend searching for best practices for deep linking on your platform (iOS or Android) and using our specific URL parameters to build your integration.

Please let me know if you have any other questions!

**Comment by adamdelphantom** [comment, marked as answer]
@BebekBebek99 I should also say we will consider adding some code examples in the future. But it may be some time before we can prioritize this. Happy to answer specific questions in the mean time if you hit a dead end.
